src/components/elements/AccessibilityDetailField.jsx

```diff
@@ -15,13 +15,31 @@ function AccessibilityDetailField({ value, onChange }) {
   }
 
   return (
-    <>
-      <label className="block font-bold mb-2">Accessibility Options</label>
-
-      <div className="grid grid-cols-3 bg-white rounded p-6">
-        {ACCESSIBILITY_OPTIONS.map(option => {
+    <div className="grid md:grid-cols-2 bg-white dark:bg-transparent rounded p-6">
+      <section>
+        {ACCESSIBILITY_OPTIONS.slice(0, ACCESSIBILITY_OPTIONS.length/2).map((option) => {
```

Comment:
Not blocking, but wondering if it would be more clear if we moved these into variables. 

```JavaScript
const sectionOneOptions = ACCESSIBILITY_OPTIONS.slice(0, ACCESSIBILITY_OPTIONS.length/2);
const sectionTwoOptions = ACCESSIBILITY_OPTIONS.slice((ACCESSIBILITY_OPTIONS.length/2), ACCESSIBILITY_OPTIONS.length);

{sectionOneOptions.map((option) => { ...
{sectionTwoOptions.mpa((option)=> { ..
```

Reply:
fixed